core/services/chainlink/testdata/config-full.toml

```diff
@@ -0,0 +1,306 @@
+Dev = true
+ExplorerURL = 'http://explorer.url'
+InsecureFastScrypt = true
+ReaperExpiration = '168h0m0s'
+RootDir = 'test/root/dir'
+ShutdownGracePeriod = '10s'
+
+[Feature]
+FeedsManager = true
+LogPoller = true
+OffchainReporting2 = true
+OffchainReporting = true
+
+[Database]
+DefaultIdleInTxSessionTimeout = '1m0s'
+DefaultLockTimeout = '1h0m0s'
+DefaultQueryTimeout = '1s'
+MigrateOnStartup = true
+ORMMaxIdleConns = 7
+ORMMaxOpenConns = 13
+
+[Database.Backup]
+Dir = 'test/backup/dir'
+Frequency = '1h0m0s'
+Mode = 'full'
+OnVersionUpgrade = true
+URL = 'http://test.back.up/fake'
```

Comment:
@samsondav Should this be moved to `secrets.toml`?

Reply:
Can follow up after this PR, but WDYT @samsondav ?

Reply:
Darn, yes, I think it probably should.